google/resource_compute_instance.go

```diff
@@ -1175,6 +1180,7 @@ func resourceComputeInstanceRead(d *schema.ResourceData, meta interface{}) error
 	d.Set("cpu_platform", instance.CpuPlatform)
 	d.Set("min_cpu_platform", instance.MinCpuPlatform)
 	d.Set("self_link", ConvertSelfLinkToV1(instance.SelfLink))
+	d.Set("instance_id", fmt.Sprintf("%d", instance.Id))
```

Comment:
Have you considered using https://golang.org/pkg/strconv/#Itoa

Reply:
I almost did that, but `Itoa` takes an `int` and `instance.Id` is of type int64.